src/pages/learn.tsx

```diff
@@ -43,6 +44,14 @@ type Props = {
   location: Location;
 }
 
+let prev = 0;
+function openNav() {
```

Comment:
Self note: put this logic into the component method so we don't make React angry.